class Rhaptos2Error(Exception):
    pass


class Rhaptos2SecurityError(Exception):
    pass
